2-structured-data/config/settings.yml.dist

```diff
@@ -2,15 +2,22 @@
 google_client_id:      YOUR_CLIENT_ID
 google_client_secret:  YOUR_CLIENT_SECRET
 google_project_id:     YOUR_PROJECT_ID
+cloudsql_connection_name: "INSTANCE_CONNECTION_NAME"
 
-# options are "cloudsql", "mongodb", or "datastore"
-bookshelf_backend: cloudsql
+# options are "cloudsql-mysql", "cloudsql-postgres", "mongodb", or "datastore"
+bookshelf_backend: cloudsql-mysql
 
-# configure CloudSQL backend
-mysql_dsn: "mysql:host=IP_ADDRESS;port=3306;dbname=YOUR_DB_NAME"
+# configure CloudSQL for MySQL backend
+mysql_dsn_local: "mysql:host=127.0.0.1;port=3306;dbname=YOUR_DB_NAME"
```

Comment:
This all looks great! One thing that is a little confusing is that `cloudsql-mysql` with `mysql_dsn_local` actually has nothing to do with CloudSQL.

One way we could fix this would be to rename a few things
 - `Datamodel\CloudSql => Datamodel\Sql` 
 - `cloudsql-mysql => mysql` 
 - `cloudsql-postgres => postgres` 
 - `mysql_dsn_deployed => mysql_dsn_cloudsql`